src/components/Apps/AppDetail/AppDetail.tsx

```diff
@@ -65,6 +68,12 @@ const AppDetail: React.FC = () => {
     }
   };
 
+  const user = useSelector(getLoggedInUser);
+  let AppInstallComponent = InstallAppModal;
+  if (user?.type === LoggedInUserTypes.MAPI) {
+    AppInstallComponent = AppInstallModal;
+  }
```

Comment:
We're mounting one or the other app installation components based on if the user is logged in via SSO or not.